### ISSUE TITLE
Create prush size

### ISSUE THREAD

**Issue by DmytroTsurkan** [BennyKok/PxerStudio#4]
In this project 
`picBoundary.set(0, 0, pxerSize * picWidth , pxerSize * picHeight );`    - like a pixel
is the brush.
When we scale this picBoundary like (pxerSize * picWidth * 2 , pxerSize * picHeight * 2)
the brush will scale too. 
But all pixels draw again and scale. Have some idea?

**Comment by BennyKok** [comment]
I am not sure if I understand correctly, did you mean you want to create a different brush size for the normal drawing tool?

**Comment by DmytroTsurkan** [comment]
Yes, you understand right. I want create different brush size, like 2x2 pixels or 3x3. And changes it dinamicaly in seek bar.

**Comment by BennyKok** [comment]
I think I did something similar for the eraser tool, which erases a brush of pixel at a time, I think you can look around that code maybe its easily convertible

**Comment by enxio27** [comment]
Need to return the default size for the eraser to 1x1 pixel, and give the user a way to set the size.

**Comment by enxio27** [comment]
> Need to return the default size for the eraser to 1x1 pixel, and give the user a way to set the size.

Do I need to open this as a new issue? The default for the eraser needs to be changed back to 1x1 pixel.  I don't have the ability to change the code and recompile it.

**Comment by BennyKok** [comment]
@enxio27 it's fine to keep it here, think we can consider changing the brush size as a similar features

> > Need to return the default size for the eraser to 1x1 pixel, and give the user a way to set the size.
> 
> Do I need to open this as a new issue? The default for the eraser needs to be changed back to 1x1 pixel. I don't have the ability to change the code and recompile it.
